src/python/pants/backend/docker/util_rules/docker_binary.py

```diff
@@ -58,14 +59,17 @@ def build_image(
             ),
             env=env,
             input_digest=digest,
+            cache_scope=ProcessCacheScope.PER_SESSION,
         )
 
     def push_image(self, tags: tuple[str, ...]) -> Process | None:
         if not tags:
             return None
 
         return Process(
-            argv=(self.path, "push", *tags), description="Pushing docker image {tags[0]}"
+            argv=(self.path, "push", *tags),
+            cache_scope=ProcessCacheScope.PER_SESSION,
+            description=f"Pushing docker image {tags[0]}",
```

Comment:
The added `test_docker_binary_push_image` test actually found an unexpected typo bug here, for the description text :P 
Even the smallest methods can hold bugs, as it were..

Reply:
Oh cool! To clarify, is this PR good to go? Is it a new typo that needs to be cherry-picked?

Reply:
No, all good. Just pointed it out, as it is being fixed in this PR (and the cherry picked one, naturally) :)